kubernetes/crd/domain-crd.yaml

```diff
@@ -26,10 +26,63 @@ spec:
           type: object
           description: DomainSpec is a description of a domain.
           properties:
+            configuration:
+              type: object
+              description: Models and overrides affecting the WebLogic domain configuration
+              properties:
+                opss:
+                  type: object
+                  description: Configuration for OPSS based security
+                  properties:
+                    keyWalletConfigMap:
+                      type: string
+                      description: The name of the config map containing the OPSS
+                        key wallet file
+                    walletSecret:
```

Comment:
I think we agreed to change this to walletFileSecret and walletPasswordSecret per slack thread https://proddev-paas-fmw.slack.com/archives/CKFA73SNM/p1580504991015300  - like other secrets both should just refer to the Name (be of type string) - unless you think it's important to have a namespace prop for secrets?  If so, then all secret references should have a namespace, not just this one...